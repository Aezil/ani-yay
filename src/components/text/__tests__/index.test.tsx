import { render } from '@testing-library/react';
import React from 'react';

import { Text } from '../index';

describe('Ani-Yay Text', () => {
  it('should render the text component', () => {
    const { container } = render(<Text>Snake?? Snake??? SNAAAAAAAAAAAAAAAAAKKKKKKKKEEEEEEEE</Text>);
    expect(container).toMatchSnapshot();
  });
});
