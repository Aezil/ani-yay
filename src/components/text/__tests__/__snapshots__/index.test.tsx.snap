// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Ani-Yay Text should render the text component 1`] = `
<div>
  <p
    class="sc-bczRLJ iPHaNS"
  >
    Snake?? Snake??? SNAAAAAAAAAAAAAAAAAKKKKKKKKEEEEEEEE
  </p>
</div>
`;
